plugins/typescript/src/core/schemaToTypeAliasDeclaration.test.ts

```diff
@@ -946,6 +946,30 @@ describe("schemaToTypeAliasDeclaration", () => {
       `);
     });
 
+    it("should generate valid identifier from number name", () => {
+      const schema: SchemaObject = {};
+
+      expect(printSchema(schema, "200")).toMatchInlineSnapshot(`
+        "export type _200 = void;"
+      `);
+    });
```

Comment:
For the case where it’s only numbers, we could use this https://github.com/fabien0102/openapi-codegen/blob/c555865557025bb69bf9d5e0dc589a965829ae4f/plugins/typescript/src/utils/getEnumProperties.ts#L95